upstream/CVE-2020-8565.json

```diff
@@ -0,0 +1,130 @@
+{
+  "id": "CVE-2020-8565",
+  "modified": "2020-10-15T22:05:32Z",
+  "published": "2020-10-15T22:05:32Z",
+  "summary": "Incomplete fix for CVE-2019-11250 allows for token leak in logs when logLevel \u003e= 9",
+  "details": "In Kubernetes, if the logging level is set to at least 9, authorization and bearer tokens will be written to log files. This can occur both in API server logs and client tool output like kubectl. This affects \u003c= v1.19.3, \u003c= v1.18.10, \u003c= v1.17.13, \u003c v1.20.0-alpha2.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/client-go"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.0/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "1.19.0"
```

Comment:
Should it be `0.19.0`?
https://github.com/aquasecurity/vuln-list-k8s/pull/83/files#r1343728478

Reply:
maybe we should remove client go as we can't find the component anyway, wdyt ?

Reply:
OK

Reply:
@knqyf263  removed !